packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.js

```diff
@@ -86,6 +88,7 @@ export default class QuanticRecentQueriesList extends LightningElement {
       },
     });
     this.unsubscribe = this.recentQueriesList.subscribe(() => this.updateState());
+    this.showPlaceholder = false;
```

Comment:
I think you should wait for `updateState()` to be called before setting `this.showPlaceholder` to `false`.